Fix compilation on ppc64le, where PATH_MAX is provided only if linux/limits.h
is included directly.

--- a/TheForceEngine/TFE_FileSystem/fileutil-posix.cpp
+++ b/TheForceEngine/TFE_FileSystem/fileutil-posix.cpp
@@ -8,6 +8,7 @@
 #include <strings.h>
 #include <sys/stat.h>
 #include <unistd.h>
+#include <linux/limits.h>
 #include <TFE_System/system.h>
 #include "fileutil.h"
 #include "filestream.h"
